ci/cloudbuild/builds/publish-docs.sh

```diff
@@ -103,40 +103,49 @@ export LANG=C.UTF-8
 
 # Stage documentation in DocFX format for processing. go/cloud-rad for details.
 function stage_docfx() {
-  local package="$1"
-  local docfx_dir="$2"
+  local feature="$1"
+  local bucket="$2"
+  local binary_dir="$3"
+  local log="$4"
+  local package="google-cloud-${feature}"
+  local path="${binary_dir}/google/cloud/${feature}/docfx"
+  if [[ "${feature}" == "common" ]]; then
+    path="${binary_dir}/google/cloud/docfx"
+  fi
 
-  if [[ ! -d "${docfx_dir}" ]]; then
-    io::log_red "Directory not found: ${docfx_dir}, skipping"
+  echo "path=${path}" >"${log}"
+  if [[ ! -d "${path}" ]]; then
+    echo "Directory not found: ${path}, skipping" >>"${log}"
+    echo "SUCCESS" >>"${log}"
     return 0
   fi
 
-  io::log "docfx_dir=${docfx_dir}"
-
-  env -C "${docfx_dir}" "${PROJECT_ROOT}/ci/retry-command.sh" 3 120 \
-    python3 -m docuploader upload \
-    --staging-bucket "${docfx_bucket}" \
-    --destination-prefix docfx .
+  if env -C "${path}" python3 -m docuploader upload \
+    --staging-bucket "${bucket}" \
+    --destination-prefix docfx . >>"${log}" 2>&1 </dev/null; then
+    echo "SUCCESS" >>"${log}"
+  fi
 }
-
-exit_status=0
+export -f stage_docfx # enables this function to be called from a subshell
 
 io::log_h2 "Publishing DocFX"
 io::log "branch:  ${BRANCH_NAME}"
 io::log "bucket:  gs://${docfx_bucket}"
 
+echo common "${FEATURE_LIST[@]}" | xargs -P "$(nproc)" -n 1 \
+  bash -c "TIMEFORMAT=\"\${0} completed in %0lR\";
+           time ci/retry-command.sh 3 120 stage_docfx \"\${0}\" \"${docfx_bucket}\" cmake-out \"cmake-out/\${0}.docfx.log\""
+
+errors=0
 for feature in common "${FEATURE_LIST[@]}"; do
-  if [[ "${feature}" == "experimental-storage-grpc" ]]; then continue; fi
-  if [[ "${feature}" == "grafeas" ]]; then continue; fi
-  if [[ "${feature}" == "experimental-opentelemetry" ]]; then feature="opentelemetry"; fi
```

Comment:
Don't we still want these?

Reply:
I think every time we loop over `FEATURE_LIST` we want to
- skip `experimental-storage-grpc`
- skip `grafeas`
- transform `experimental-opentelemetry` -> `opentelemetry`

Reply:
I fixed `experimental-opentelemetry` -> `opentelemetry`. The other changes are not needed for the reasons explained in the new comments.